cmd/minikube/cmd/start_flags.go

```diff
@@ -196,6 +197,11 @@ func initDriverFlags() {
 	startCmd.Flags().String(hypervVirtualSwitch, "", "The hyperv virtual switch name. Defaults to first found. (hyperv driver only)")
 	startCmd.Flags().Bool(hypervUseExternalSwitch, false, "Whether to use external switch over Default Switch if virtual switch not explicitly specified. (hyperv driver only)")
 	startCmd.Flags().String(hypervExternalAdapter, "", "External Adapter on which external switch will be created if no external switch is found. (hyperv driver only)")
```

Comment:
please add a check inside  validateFlags 
to ensure that if user provided this flag with a driver other than Docker it should tell the user this flag wont' work with their driver and they should use --mount-string instead for VM drivers.

Reply:
Thank you! I did